lib/action_subscriber/babou.rb

```diff
@@ -9,8 +11,11 @@ def self.start_subscribers
       ::ActionSubscriber.setup_subscriptions!
       ::ActionSubscriber.print_subscriptions
       ::ActionSubscriber.start_subscribers!
-      logger.info "Action Subscriber connected"
 
+      ::FileUtils.mkdir_p(File.join(Rails.root, "tmp"))
+      ::FileUtils.touch(File.join(Rails.root, "tmp", "subscriber-started"))
```

Comment:
Seeing this I am realizing that we probably shouldn't make Rails a hard requirement here since it's not technically required to run action subscriber. How can we move this to the configuration.rb and railtie.rb to make this a generic feature? Maybe we should use active support instrumentation and add a callback in lib/action_subscriber.rb similar to how ruby-protobuf solves this problem?

Reply:
Could even be as simple as:
```
::ActiveSupport::Notifications.instrument("action_subscriber:server_started")

# lib/action_subscriber.rb
class ActionSubscriber
# ...
  def self.after_server_start(&block)
    ::ActiveSupport::Notifications.subscribe("action_subscriber:server_started") do |*args|
      block.call(*args)
    end
  end
# ...
end
```

Reply:
Ok.  Pretty similar to proto.  First part of the comment was to move away from rails.root right ?